enos/modules/target_set_up_external_integration/main.tf

```diff
@@ -0,0 +1,70 @@
+# Copyright (c) HashiCorp, Inc.
+# SPDX-License-Identifier: BUSL-1.1
+
+terraform {
+  required_providers {
+    enos = {
+      source = "registry.terraform.io/hashicorp-forge/enos"
+    }
+  }
+}
+
+locals {
+  distro              = var.distro
+  test_server_address = var.ip_version == "6" ? var.hosts[0].ipv6 : var.hosts[0].public_ip
+  ldap_server = {
+    domain     = "enos.com"
+    org        = "hashicorp"
+    admin_pw   = "password1"
+    version    = var.ldap_version
+    port       = "389"
```

Comment:
Rather than implicitly expect 389 and 636 as ports for LDAP, rely on the value being passed in from our port map in enos-globals.

I.e. We should make the LDAP and LDAPS ports variables to module and pass in the value either in the step or set them as defaults in the module definition.